Source/Views/PageView.swift

```diff
@@ -138,9 +138,8 @@ class PageView: UIScrollView {
     let newZoomScale = zoomScale > minimumZoomScale
       ? minimumZoomScale
       : maximumZoomScale
-
-    let width = contentFrame.size.width / newZoomScale
-    let height = contentFrame.size.height / newZoomScale
+    let width = imageView.frame.width / newZoomScale
+    let height = imageView.frame.height / newZoomScale
```

Comment:
We're zooming the imageView (which is smaller than the contentFrame unless the image happens to be the exact same size). This caused weirdness, especially with <1 zoom values that we're now using.